scripts/analytics/file_summary.py

```diff
@@ -0,0 +1,77 @@
+import django
+django.setup()
+
+from django.db.models import Q
+import logging
+from dateutil.parser import parse
+from datetime import datetime, timedelta
+from django.utils import timezone
+
+from website.app import init_app
+from scripts.analytics.base import SummaryAnalytics
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+class FileSummary(SummaryAnalytics):
+
+    @property
+    def collection_name(self):
+        return 'file_summary'
+
+    def get_events(self, date):
+        super(FileSummary, self).get_events(date)
+        from addons.osfstorage.models import OsfStorageFile
+
+        # Convert to a datetime at midnight for queries and the timestamp
+        timestamp_datetime = datetime(date.year, date.month, date.day).replace(tzinfo=timezone.utc)
+        query_datetime = timestamp_datetime + timedelta(days=1)
+
+        file_qs = OsfStorageFile.objects
```

Comment:
Nit (not a blocker, just commenting for awareness): `file_qs` is _mildly_ misleading since `OsfStorageFile.objects` returns an instance of the manager, whereas `OsfStorageFile.objects.filter()` returns a queryset.

```
In [11]: type(OsfStorageFile.objects)
Out[11]: osf.models.files.BaseFileNodeManager

In [12]: type(OsfStorageFile.objects.filter())
Out[12]: include.query.IncludeQuerySet
```